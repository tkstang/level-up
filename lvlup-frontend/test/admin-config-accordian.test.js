import ConfigurationAccordian from '../src/components/Admin/admin-config/config-accordian';
import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

test('ConfigurationAccordian should render', () => {
  const config = shallow(
    <ConfigurationAccordian />,
  );
  expect(toJson(config)).toMatchSnapshot();
});
