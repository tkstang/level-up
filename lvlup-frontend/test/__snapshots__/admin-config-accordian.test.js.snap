exports[`test ConfigurationAccordian should render 1`] = `
<Accordion
  exclusive={false}
  fluid={true}
  panels={
    Array [
      Object {
        "title": 0,
      },
      Object {
        "title": 1,
      },
      Object {
        "title": 2,
      },
    ]
  } />
`;
